frontend/src/tests/ddah-tests.js

```diff
@@ -272,20 +272,21 @@ export function ddahTests(api) {
         expect(resp).toHaveStatus("success");
         expect(resp.payload.approved_date).toBeFalsy();
 
-        // Updating a signed DDAH should remove the signature
-        resp = await apiPOST(`/admin/ddahs`, ddah);
+        // Sign the DDAH
+        resp = await apiPOST(
+            `/public/ddahs/${ddah.url_token}/accept`,
+            { signature: "My Sig" },
+            true
+        );
         expect(resp).toHaveStatus("success");
-        ddah = resp.payload;
-        resp = await apiPOST(`/admin/ddahs`, {
-            ...ddah,
-            signature: "My Sig",
-            accepted_date: new Date(),
-        });
+        // Get the signed DDAH since it is not returned from public route
+        resp = await apiGET(`/admin/assignments/${newAssignment.id}/ddah`);
```

Comment:
I suppose the reason that the `/public` route does not return the newly updated record is security-related?